app/src/main/java/com/hammer/talkbbokki/presentation/comment/CommentDetailScreen.kt

```diff
@@ -0,0 +1,244 @@
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.hammer.talkbbokki.R
+import com.hammer.talkbbokki.ui.theme.*
+
+@Composable
+fun CommentDetailRoute(
+    onBackClick: () -> Unit,
+    viewModel: CommentsViewModel = hiltViewModel()
+) {
+    val recomments = viewModel.commentItems.collectAsState()
+    CommentDetailScreen(
+        onBackClick,
+        recomments.value.first(),
+        recomments.value
+    )
+}
+
+@Composable
+fun CommentDetailScreen(
+    onBackClick: () -> Unit,
+    comment: Comment,
+    recomments: List<Comment>
+) {
+    Box(
+        Modifier
+            .fillMaxSize()
+            .background(MainBackgroundColor)
+    ) {
+        Column(
+            Modifier
+                .fillMaxSize()
+                .padding(bottom = 78.dp)
+        ) {
+            CommentDetailHeader(onBackClick)
+            CommentContents(
+                nickname = comment.nickname,
+                date = comment.date,
+                content = comment.content,
+                replyCount = comment.replyCount,
+                onDeleteClick = comment.onDeleteClick
+            )
+            RecommentList(
+                recomments
+            )
+        }
+        Box(Modifier.align(Alignment.BottomCenter)) {
+            RecommentInputArea()
+        }
+    }
+}
+
+@Composable
+fun CommentDetailHeader(
+    onBackClick: () -> Unit,
+) {
+    val replyCount = 0
+    Box(
+        Modifier
+            .height(56.dp)
+            .fillMaxWidth()
+    ) {
+        Icon(painter = painterResource(id = R.drawable.ic_arrow_left),
+            tint = White,
+            contentDescription = null,
+            modifier = Modifier
+                .clickable {
+                    onBackClick()
+                }
+                .align(Alignment.CenterStart)
+                .padding(start = 20.dp))
+        Text(
+            text = "답글 ($replyCount)", Modifier.align(Alignment.Center), color = White
+        )
+    }
+}
+
+@Composable
+fun RecommentItem(
+    nickname: String, date: String, content: String, replyCount: Int, onDeleteClick: () -> Unit
+) {
+    Box(modifier = Modifier.padding(20.dp)) {
+        Image(
+            modifier = Modifier.size(18.dp),
+            painter = painterResource(id = R.drawable.ic_recomment),
+            contentDescription = null
+        )
+        CommentContents(
+            nickname = nickname,
+            date = date,
+            content = content,
+            replyCount = replyCount,
+            onDeleteClick = onDeleteClick
+        )
+    }
+}
+
+@Composable
+fun CommentContents(
+    nickname: String, date: String, content: String, replyCount: Int, onDeleteClick: () -> Unit
```

Comment:
🚫 **[ktlint]** <standard:parameter-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter should start on a newline